layouts/partials/body_text.html

```diff
@@ -0,0 +1,10 @@
+{{ if .md }}
+  {{ $md_body := .context.body | markdownify }}
+  {{ if not (in $md_body "<p>") }}
+    {{ printf "<p>%s</p>" $md_body }}
```

Comment:
When I tested this, it seems that the output needs to be piped through `safeHTML`, otherwise the rendered HTML will be escaped:

```suggestion
    {{ printf "<p>%s</p>" $md_body | safeHTML }}
```